### PR TITLE
DOCS-6917 fix proxy domain doc errors

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
 = Domain Support for Mule 4 API Proxies
 
-Mule runtime engine enables you to share common resources among other Mule applications by defining these resources within a Mule domain. Other Mule applications can then inherit this domain.
+Mule runtime engine (Mule) enables you to share common resources among other Mule applications by defining these resources within a Mule domain. Other Mule applications can then inherit this domain.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: delete "other" (among Mule applications).

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 This way, your Mule applications that are configured for it, can access the xref:4.1@mule-runtime::shared-resources.adoc[resources]declared in your domain. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: "it" has no clear antecedent. I'm not sure what "This way" or "it" refers to.

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
-Because every auto-generated proxy is a preconfigured Mule application, you can also define multiple API proxies under the same domain. You install a domain when you auto-deploy multiple proxies that use the same host and port but have a different path. +
+Because every auto-generated proxy is a preconfigured Mule application, you can define multiple API proxies under the same domain. You create a domain when you auto-deploy multiple proxies that use the same host and port but have a different path. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: I don't understand the 2nd sentence very clearly. Is it saying that the only way to create a domain is to auto-deploy multiple proxies that use the same host & port but have a different path (do we mean different endpoints? Base URLs?

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 4]
```diff
@@ -15,16 +15,15 @@ API Manager enables you to automatically configure an API proxy to inherit an ex
 
 Note that API domains are not supported by Runtime Fabric or CloudHub support API domains. 
 
-*Important:* In an API domain, you may define several dependencies. Ensure that the versions of the dependencies defined in the domain have the same or later version than the ones defined in the API proxy that references the domain.
+*Important:* You may define several dependencies in an API domain. Ensure that the versions of the dependencies defined in the domain have the same or later version than the ones defined in the API proxy that references the domain.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: Is an API domain different from a proxy domain? The same (just for a proxy or an API)?

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 5]
```diff
@@ -15,16 +15,15 @@ API Manager enables you to automatically configure an API proxy to inherit an ex
...
 
-All the dependencies defined in the domain override the dependencies defined in the API proxy (older or newer). For example, if the API proxy depends on a functionality defined in the HTTP transport, but is referencing a domain using
- an older version of that transport, you will have errors at runtime.
+All the dependencies defined in the domain override the dependencies defined in the API proxy (older or newer). For example, if the API proxy depends on a functionality defined in the HTTP transport, but is referencing a domain using an older version of that transport, you will have errors at runtime.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: "you will have errors" should remove future tense and 2nd person pronoun. Something like "errors are thrown at runtime" or "the proxy throws errors at runtime".

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 6]
```diff
@@ -42,15 +41,18 @@ For HTTP listener configuration defined in the domain, the listener setup on the
...
+..  To import the API from Exchange, Select *Manage API from Exchange*.
+.. To import the API from your local machine, select *Import*.
 . Specify an API name, Asset type, API version, and Asset version.
 . In *Managing Type*, select *Endpoint with Proxy*, and then in *Proxy deployment target*, select *Hybrid*.
 . In *Mule Version*, select the *Check this box if you are managing this API in Mule 4 or above* checkbox.
 . In *Implementation URI*, type your API URI. 
-. Additionally specify:
-.. A TLS Context, which is required if you plan to have xref:building-https-proxy.adoc[HTTPs] communications. 
-.. The path for your implementation URI if it is different than `/`.
+. If you plan to have xref:building-https-proxy.adoc[HTTPs] communications, specify a TLS Context.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: [HTTPS]

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 7]
```diff
@@ -59,23 +61,28 @@ To configure an API proxy to use a domain:
...
+. After saving the API proxy configuration, perform one of the following steps:
 
 * Deploy the API proxy to the selected target (*Settings*, under *Deployment Configuration*).
 * Download the API proxy (*Actions*) and edit it in Anypoint Studio to customize your target domain before deploying it.
 
 == Configuring a Domain for the First Time
 
+When referencing a domain, Mule reads the domain Maven coordinates from the API proxy `pom.xml` file and looks for an already deployed domain with the same semantic name. For example, in the case of an API gateway domain with the version specified in the previous example, the semantic name is: `gateway-proxy-domain-1.0.0-mule-domain`.
+
+For a proxy to leverage the domain, you must ensure that the domain template is specified in the API proxy `pom.xml` as a provided dependency.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: delete "you must"

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 8]
```diff
@@ -59,23 +61,28 @@ To configure an API proxy to use a domain:
...
+When referencing a domain, Mule reads the domain Maven coordinates from the API proxy `pom.xml` file and looks for an already deployed domain with the same semantic name. For example, in the case of an API gateway domain with the version specified in the previous example, the semantic name is: `gateway-proxy-domain-1.0.0-mule-domain`.
+
+For a proxy to leverage the domain, you must ensure that the domain template is specified in the API proxy `pom.xml` as a provided dependency. 
+
 To configure a domain for the first time:
 
 . Obtain a domain artifact in one of the following ways:
 .. Download the https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API Gateway domain template].
-.. Develop your own custom domain, using the naming convention predefined in the autogeneated proxy: +
+.. Develop your own custom domain, using the naming convention predefined in the auto-generated proxy: +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: autogenerated per CMS & Merriam Webster: https://www.chicagomanualofstyle.org/book/ed17/part2/ch07/psec089.html

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 9]
```diff
@@ -59,23 +61,28 @@ To configure an API proxy to use a domain:
...
 ** `api-proxy-listener-http`: The listener configuration for HTTP communications 
 ** `api-proxy-listener-https`: The listener configuration for HTTPS communications
 +
-. Copy the Maven artifact to `../domains/` folder of your running runtime distribution, (for example, `gateway-proxy-domain-1.0.0-mule-application-template.jar`).+
-*Tip:* Ensure that the name of the artifact and its Maven coordinates match:
+. Copy the Maven artifact to `../domains/` folder of your running runtime distribution, for example, `gateway-proxy-domain-1.0.0-mule-application-template.jar`.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: "to the `../domains/` folder" or " to `../domains/` "

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 10]
```diff
@@ -110,18 +111,18 @@ To avoid a conflict when running multiple proxies using the same domain, ensure
 
 === API Gateway Domain Template
 
-If you choose to download your API proxy and configure it manually on your end, you may use the *API Gateway domain template*.
+If you choose to download and manually configure your API proxy on your end, use the *API Gateway domain template*.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: delete "on your end" (unless the reader also has an api proxy to manually configure on some other end).

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 11]
```diff
@@ -110,18 +111,18 @@ To avoid a conflict when running multiple proxies using the same domain, ensure
...
 
-The https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API Gateway domain template] is configured to have a shared HTTP listener configuration ("api-proxy-listener-http") listening on the 8081 port. You can also have a shared HTTPs listener configuration ("api-proxy-listener-https"). 
+The https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API gateway domain template] is configured to have a shared HTTP listener configuration ("api-proxy-listener-http") listening on the 8081 port. You can also have a shared HTTPs listener configuration ("api-proxy-listener-https").
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: HTTPS

File: modules/ROOT/pages/proxy-domain-support.adoc [hunk 12]
```diff
@@ -110,18 +111,18 @@ To avoid a conflict when running multiple proxies using the same domain, ensure
...
 
-The https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API Gateway domain template] is configured to have a shared HTTP listener configuration ("api-proxy-listener-http") listening on the 8081 port. You can also have a shared HTTPs listener configuration ("api-proxy-listener-https"). 
+The https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API gateway domain template] is configured to have a shared HTTP listener configuration ("api-proxy-listener-http") listening on the 8081 port. You can also have a shared HTTPs listener configuration ("api-proxy-listener-https"). 
 
-To use either configuraton, you must uncomment code from your API Gateway domain template and configure your TLS context, such as, certificates and passwords. The following list provides the available listeners configurations:
+To use either configuration, you must uncomment code from your API gateway domain template and configure your TLS context, such as, certificates and passwords. The following list provides the available listeners configurations:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/proxy-domain-support.adoc]: delete "you must"
Unless we tell them something is optional, every instruction is required. Saying only some of them are "must" is inconsistent.

Remove the comma after "such as"

**Review by Mulesoft-Mysti — APPROVED** [review]

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: path is a field in the UI and it is part of creating a proxy domain. Thanks!

**Review by JaatadiaMulesoft — APPROVED** [review]

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Changed to autogenerated! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 11 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Great catch! Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 12 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

File: modules/ROOT/pages/proxy-domain-support.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/proxy-domain-support.adoc]: Done! Thanks!

**Pull request merged**